app.Makefile

```diff
@@ -29,13 +30,82 @@ $(shell echo '$(APP_PARAMETERS)' '$(APP_TEST_PARAMETERS)' \
 endef
 
 
+##### Tools development targets #####
+
+
 .build/app: | .build
 	mkdir -p "$@"
 
 
+.build/app/base_deployer: | .build/app
+	mkdir -p "$@"
+
+
+##### Tools development targets #####
```

Comment:
this is aimed at making it convenient for users to leverage `marketplace-k8s-app-tools`'s build system, but I thought we wanted to reduce those dependencies.

ideally i think partners will not build any of our assets, and we're not far from that goal - what do you think?

Reply:
We would not encourage partners to use our Makefiles anymore. I am aware that several partners are already using them, so unless we have to, we don't want to kill them right now.

We still have dependencies on these Makefiles from example and c2d, and currently they don't work for local development. This is to address that broken workflow.

I don't mind deleting them all in the future. We need to figure out what we want to do at that time. I have a hunch that simply replacing Makefiles with bash scripts is not gonna be better.

Reply:
i worry that exposing adding local development targets to `app.Makefile` exposes our build implementation more broadly than necessary.

what do you think about adding this functionality to `marketplace.Makefile` instead?